### PR TITLE
[photos] Update for iOS 10 beta 1

### PR DESCRIPTION
**Pull request by spouliot**
Also enable photos.framework on tvOS and macOS

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by spouliot** [comment]
build

only the mac side was tested, without errors, previously  (bot issue)

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by spouliot** [comment]
@chamons since you're running on Sierra can you tell me if `/System/Library/Frameworks/Photos.framework/Photos` is 64 bits only or a fat library ? I'll fix apitest and, if needed, the `[Mac]` attributes. Thanks!

**Comment by chamons** [comment]
@spouliot x86_64 only

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by spouliot** [comment]
build

**Comment by xamarin-release-manager** [comment]
Build success

File: src/photos.cs [hunk 1]
```diff
@@ -953,4 +1056,83 @@ interface PHLivePhotoInfo {
...
+		[Abstract]
+		[Export ("type")]
+		PHLivePhotoFrameType Type { get; }
+
+		[Abstract]
+		[Export ("renderScale")]
+		nfloat RenderScale { get; }
+	}
+
+#if false // returns null on iOS 10 beta 1
```
Comment (dalexsoto) [inline src/photos.cs]: Maybe add the `TODO:` or `"FIXME` so it is easier to spot later

**Comment by dalexsoto** [comment]
LGTM 👍

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**